src/libs/ajax/GoogleStorage.ts

```diff
@@ -1,15 +1,58 @@
 import _ from 'lodash/fp'
 import * as qs from 'qs'
-import { fetchBuckets, saToken } from 'src/libs/ajax'
-import { authOpts, fetchOk, jsonBody } from 'src/libs/ajax/ajax-common'
+import { saToken } from 'src/libs/ajax'
+import { authOpts, fetchOk, jsonBody, withRetryOnError, withUrlPrefix } from 'src/libs/ajax/ajax-common'
+import { withRequesterPays } from 'src/libs/ajax/Billing'
```

Comment:
this little guy is the main thing worth paying attention to with the movement of the `Billing` module. 

Essentially, we had a fairly nasty dependency to untangle where Billing module and GoogleStorage need to make the same API calls about buckets. This is expressed far better with this line. Essentially, Billing holds some common methods that require `GoogleStorage` to depend on it.